sceneview_1_0_0/build.gradle

```diff
@@ -115,9 +115,6 @@ dependencies {
     api "androidx.fragment:fragment-ktx:1.5.5"
     api "androidx.lifecycle:lifecycle-runtime-ktx:2.5.1"
 
-    // Android Material
-    implementation 'com.google.android.material:material:1.7.0'
```

Comment:
Shouldn't it be removed from the sceneview (pre-1.0.0) module as well?

Reply:
You are right

Reply:
I'll try to remove it (there was a dependency in the `LoadingNode` class).

Reply:
I've removed the remaining dependencies and the `LoadingNode` class.